### ISSUE TITLE
[Bug] <The addons do not apply to the game suddenly>

### ISSUE THREAD

**Issue by tirredtony** [BiscuitDevelopment/SkyblockAddons#485]
Please note that we do not provide support for clients other than Forge.
We will not fix compatibility issues with other clients.

**Describe the bug**
when i launch minecraft and go in hypixel skyblock the addons simply don't apply on the game for exemple even if the lock is activated you can't lock things and also many more as the hp bar and mana bar, etc...

**To Reproduce**
Steps to reproduce the behavior (how did you get this bug to occur):
1. Go to 'hypixel skyblock'
2. Click on 'well nothing since it doesn't work from opening the game'

**Expected behavior**
Well what is supposed to happen is the addons applying on the game and beeing able to lock slots and se my mana and hp bars

**Screenshots/Video**
![Capture d’écran 2021-01-21 212508](https://user-images.githubusercontent.com/77818266/105437559-4d000f80-5c2f-11eb-9b9b-503145ca4f8b.png)
![Capture d’écran 2021-01-21 212434](https://user-images.githubusercontent.com/77818266/105437568-4ffb0000-5c2f-11eb-9324-0a84ba36c641.png)
so basically my whole hotbar should be locked but i can not lock them.

**Crash Report**
If your game crashed, please paste your crash report below.

```text
[Paste Here]
```

**System Information**
 - OS: [e.g. Windows 10]
 - Minecraft Version: [e.g. 1.8.9]
 - Forge Version: [e.g. 11.15.1.2318]
 - SkyblockAddons Version: [e.g. 1.5.4]

**Other Installed Mods**
i had other mods but desinstalled them and tested qith only the addons and it would still not work.
**Additional info**

**Comment by Pnda1** [comment]
Hello,
Such a bug has already been reported and is due to an issue on Hypixel where they changed some things that caused SBA's playing skyblock detection to not work properly. Please join our [discord](https://discord.gg/biscuit) and see the #announcements channel for a file that fixes said bug.

**Comment by tirredtony** [comment]
> Hello,
> Such a bug has already been reported and is due to an issue on Hypixel where they changed some things that caused SBA's playing skyblock detection to not work properly. Please join our [Discord](https://discord.gg/biscuit) and see the #announcements channel for a file that fixes said bug.

ok thank you so much because badlion really isn't good compared to actual sba
